base/logger_audit.go

```diff
@@ -34,10 +34,12 @@ const (
 )
 
 // expandFields populates data with information from the id, context and additionalData.
-func expandFields(id AuditID, ctx context.Context, additionalData AuditFields) AuditFields {
-	var fields AuditFields
+func expandFields(id AuditID, ctx context.Context, globalFields AuditFields, additionalData AuditFields) AuditFields {
+	fields := auditLogger.baseFields()
 	if additionalData != nil {
-		fields = additionalData
+		for k, v := range additionalData {
+			fields[k] = v
+		}
```

Comment:
Order of precedence here is backwards. Sorry, I said I'd update the Jira ticket with details and never got around to it.

We need SG's fields to be overwritten by the injected global fields, and then overwritten by the request-specific fields.